sql/core/src/test/resources/sql-tests/inputs/ansi/try_cast.sql

```diff
@@ -0,0 +1 @@
+--IMPORT try_cast.sql
```

Comment:
I feel a bit overkill to run all the end-to-end tests again with ANSI mode. The code is very clear that `TRY_CAST` does not rely on the ANSI flag, and it's good enough to write some UTs to verify it.

Reply:
OK, let's delete it